vscode/src/extension.ts

```diff
@@ -514,17 +514,24 @@ export function activate(context: ExtensionContext): VSNetBeansAPI {
             vscode.window.showErrorMessage('Cannot find workspace path');
             return;
         }
+        
         const userDir = path.join(storagePath, "userdir");
         if (userDir && fs.existsSync(userDir)) {
-            const confirmation = await vscode.window.showInformationMessage('Are you sure you want to delete cache for this workspace?',
+            const confirmation = await vscode.window.showInformationMessage('Are you sure you want to delete cache for this workspace  and reload the window ?',
                 'Yes', 'Cancel');
             if (confirmation === 'Yes') {
-                await fs.promises.rmdir(userDir, {recursive : true});
-                const res = await vscode.window.showInformationMessage('Cache cleared successfully for this workspace', 'Reload window');
-                if (res === 'Reload window') {
-                    await vscode.commands.executeCommand('workbench.action.reloadWindow');
+                    try{
+                        await stopClient(client);
+                        deactivated=true;
+                        await killNbProcess(false,log);
+                        await fs.promises.rmdir(userDir, {recursive : true});
+                        await vscode.window.showInformationMessage("Cache deleted successfully",'Reload window');
```

Comment:
nitpick: await not required here in my opinion

Reply:
If I remove the await then the message may not be shown to the user "Cache deleted successfully" and the window will be reloaded . I tested without the await it doesn't work .

Reply:
Sorry, my bad I thought it's just a simple information message, I noticed just now it is user action information message due to that await is required. Sorry, I should have seen in a more diligent way.